src/extensions/mountainview/MountainView/ViewContainerTabBar.tsx

```diff
@@ -0,0 +1,67 @@
+import { IconButton, Tab, Tabs } from '@material-ui/core';
+import CloseIcon from "@material-ui/icons/Close";
+import { default as React, FunctionComponent, useCallback, useEffect } from 'react';
+import { View } from './MountainView';
+
+type Props = {
+    views: View[]
+    currentView: View | null
+    onCurrentViewChanged: (v: View) => void
+    onViewClosed: (v: View) => void
+}
+
+const ViewContainerTabBar: FunctionComponent<Props> = ({ views, currentView, onCurrentViewChanged, onViewClosed }) => {
+    const handleChange = useCallback((event: React.ChangeEvent<{}>, newTabIndex: number) => {
+        const v = views[newTabIndex]
+        if (v) onCurrentViewChanged(v)
+    }, [onCurrentViewChanged, views])
+    useEffect(() => {
+        const i = currentView ? views.indexOf(currentView) : -1
+        if (i < 0) {
+            if (views.length > 0) {
+                onCurrentViewChanged(views[0])
+            }
+        }
+    }, [currentView, onCurrentViewChanged, views])
+    let currentIndex: number | null = currentView ? views.indexOf(currentView) : null
+    if (currentIndex === -1) currentIndex = null
+    return (
+        <Tabs
+            value={currentIndex || 0}
+            onChange={handleChange}
+            scrollButtons="auto"
+            variant="scrollable"
+        >
+            {views.map(v => (
+                viewContainerTab(v, onViewClosed)
+            ))}
+        </Tabs>
+    )
+}
+
+const viewContainerTab = (view: View, onClose: (view: View) => void) => {
```

Comment:
This is an individual tab. There is a reason that is just a function rather than a function component. It is because <Tabs> expects the direct children to be <Tab> elements.